drivers/health_qa_factory/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+BostonHmis::Application.routes.draw do
+  scope :health_qa_factory do
+    # TODO
+    # get '/my_path', to: 'health_qa_factory/my_controller'
```

Comment:
Will these QA be exposed in the old QA view?  Does it make sense to augment that with some of the factory information that would join things together?

Reply:
They are just ordinary QAs -- for 1.0 we created them inside the `XxxSaver` classes, but I didn't like how hard it was to figure out which saver generated which QA -- so, I moved that logic out to here,